clang/include/clang/Basic/Attr.td

```diff
@@ -2189,6 +2189,14 @@ def BTFTypeTag : TypeAttr {
   let LangOpts = [COnly];
 }
 
+def BPFFastCall : InheritableAttr,
+                  TargetSpecificAttr<TargetBPF> {
+  let Spellings = [GCC<"bpf_fastcall">];
+  let Subjects = SubjectList<[FunctionLike]>;
```

Comment:
Do you have to care about Objective-C methods or C++ member functions?

Reply:
No, only focus on C at the moment.

Reply:
Excellent!